mne/viz/utils.py

```diff
@@ -2332,6 +2332,7 @@ def _setup_ax_spines(axes, vlines, tmin, tmax, invert_y=False,
         axes.yaxis.set_label_coords(-0.05, 1 - ylabel_height
                                     if invert_y else ylabel_height)
 
+    vlines = [] if vlines is None else vlines
```

Comment:
could also make vlines parameter default to `[]` in the function signature, but that would have required reordering the function params.